sqflite/lib/src/utils/sentry_database_span_attributes.dart

```diff
@@ -10,3 +10,12 @@ void setDatabaseAttributeData(ISentrySpan? span, String? dbName) {
     span?.setData(SentryDatabase.dbNameKey, dbName);
   }
 }
+
+/// Sets the database attributes on the [breadcrumb].
+/// It contains the database system and the database name.
+void setDatabaseAttributeOnBreadcrumb(Breadcrumb breadcrumb, String? dbName) {
+  breadcrumb.data?[SentryDatabase.dbSystemKey] = SentryDatabase.dbSystem;
```

Comment:
[Docs](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types) suggest to set the breadcrumb type for database operation to `query`, even if don't have that for iOS, Android or RN
Let's set it here, as this is used for all the database breadcrumbs

Reply:
Need to make a copy since the type of the breadcrumb is immutable.

Reply:
I've added the type in the ctor